src/pages/topics/Topic.tsx

```diff
@@ -109,11 +123,14 @@ const Topic: React.FC<Props> = ({ match }) => {
             </IonListHeader>
             {references.map((reference: any, index) => (
             <IonItem key={index} routerLink={"/constitution/provision/" + reference.id}>
-              <IonLabel>
-                <h3>{ reference.title }</h3>
-              </IonLabel>
+              <p>{reference.title}</p>
             </IonItem>
             ))}
+            <IonItem>
+              <div className="ion-padding">
```

Comment:
Can we put this in just after the heading, around line 102? The idea is to pull in the content above, and not (yet) replace this list.